src/components/slider/slider.st.css

```diff
@@ -78,234 +26,85 @@
     -webkit-touch-callout: none;
 }
 
-.root:x {
-    width: 100%;
-    padding: value(clickableAreaRadius) 0;
+.root:disabled {
+    pointer-events: none;
+    cursor: not-allowed;
 }
 
-.root:xReverse {
+.root:x, .root:xReverse {
     width: 100%;
-    padding: value(clickableAreaRadius) 0;
 }
-
-.root:y {
-    min-height: value(minVerticalSliderHeight);
+.root:y, .root:yReverse {
     height: 100%;
-    padding: 0 value(clickableAreaRadius);
-}
-
-.root:yReverse {
-    min-height: value(minVerticalSliderHeight);
-    height: 100%;
-    padding: 0 value(clickableAreaRadius);
 }
 
 .bar {
-    position: relative;
-    border-radius: value(trackRadius);
-    background-color: value(colorTrack);
-}
-
-.root:x .bar{
-    height: value(trackSize);
-    width: 100%;
+    position: absolute;
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+    margin:auto;
+    
 }
 
-.root:xReverse .bar{
+.root:x .bar, .root:xReverse .bar {
     height: value(trackSize);
     width: 100%;
 }
 
-.root:y .bar{
+.root:y .bar, .root:yReverse .bar {
     text-align: left;
     width: value(trackSize);
     height: 100%;
-    min-height: value(minVerticalSliderHeight);
-}
-
-.root:yReverse .bar{
-    text-align: left;
-    width: value(trackSize);
-    height: 100%;
-    min-height: value(minVerticalSliderHeight);
 }
 
 .rangeBar {
     position: absolute;
-    background-color: value(colorProgress);
-    border-radius: value(trackRadius);
 }
 
-.root:x .rangeBar{
-    left: 0;
+.root:x .rangeBar, .root:xReverse .rangeBar{
     height: inherit;
 }
 
-.root:xReverse .rangeBar{
-    right: 0;
-    height: inherit;
+.root:x .rangeBar {
+    left: 0;
 }
 
-.root:y .rangeBar{
-    bottom: 0;
-    width: inherit;
+.root:xReverse .rangeBar {
+    right: 0;
 }
 
-.root:yReverse .rangeBar{
-    top: 0;
+.root:y .rangeBar, .root:yReverse .rangeBar{
+    bottom: 0;
     width: inherit;
 }
 
 .handle {
     -st-states: active;
     position: absolute;
-    width: value(handleWidth);
-    height: value(handleHeight);
-    border-radius: value(handleRadius);
-    background-color: value(colorHandle);
-    border: solid 1px;
-    border-color: value(handleBorder);
     box-sizing: border-box;
+    z-index: 1;
 }
-.handle:hover {
-    box-shadow: none;
-    background-color: value(colorHandle_Active);
-    border-color: value(colorHandle_Active);
-}
+
 .handle:active {
-    z-index: 1;
+    z-index: 2;
 }
+
 .handle:active .tooltip {
     display: block;
 }
+
 .handle:hover .tooltip {
     display: block;
 }
 
-.root:x .handle{
-    margin: value(middleOfBar) 0 0 value(middleOfHandle);
-}
-
-.root:xReverse .handle{
-    margin: value(middleOfBar) value(middleOfHandle) 0px 0px;
-}
-
-.root:y .handle{
-    margin: 0px 0px value(middleOfHandle) value(middleOfBar);
-}
+.mark {}
 
-.root:yReverse .handle{
-    margin: value(middleOfHandle) 0px 0px value(middleOfBar);
-}
+.rangeMark {}
 
 .rangeMark,
 .mark
 {
```

Comment:
we should use one style for curly braces: or always new line, or always last selector line